crates/nu-protocol/src/errors/shell_error.rs

```diff
@@ -595,6 +595,19 @@ pub enum ShellError {
         src_span: Span,
     },
 
+    /// Not all rows contain data in the requested column.
+    ///
+    /// ## Resolution
+    ///
+    /// Use the Optional Operator, --ignore-errors with get or select, or fill in the missing values.
+    #[error("Not all rows contain a value for '{col_name}'")]
+    #[diagnostic(code(nu::shell::row_lacks_value))]
+    RowLacksValue {
+        col_name: String,
+        #[label = "try using '{col_name}?'"]
+        span: Option<Span>,
```

Comment:
This points to the cell path component (if that is set correctly in all circumstances)

Question if we should have another label pointing to the source `Value` guaranteed to have a span (even if sometimes a bit weirdly located)